app/services/usps_in_person_proofing/proofer.rb

```diff
@@ -1,5 +1,7 @@
 module UspsInPersonProofing
   class Proofer
+    mattr_reader :token
+    mattr_reader :token_expires_at
```

Comment:
Do these need to be (or is there a strong desire for these to be) on separate lines?

```suggestion
    mattr_reader :token, :token_expires_at
```